salt/kubelet/init.sls

```diff
@@ -12,10 +12,12 @@ include:
     - source: salt://kubelet/kubelet-initial.jinja
     - template: jinja
     - defaults:
-{% if not "kube-master" in salt['grains.get']('roles', []) %}
-      schedulable: "true"
+{% if "kube-master" in salt['grains.get']('roles', []) %}
+      nodeLabels: "node-role.kubernetes.io/master="
```

Comment:
Shouldn't we set that to be:
```
 nodeLabels: "node-role.kubernetes.io/master=true"
```

To simplify label matching later on (see the dex PR)

Reply:
No, for some reason I can't fathom, master=<empty string> was chosen upstream.

See https://github.com/kubernetes/kubeadm/blob/master/docs/design/design_v1.9.md#mark-master